src/coreclr/jit/emitxarch.cpp

```diff
@@ -867,91 +1081,456 @@ emitter::code_t emitter::AddRexWPrefix(instruction ins, code_t code)
             return emitter::code_t(code | 0x00008000000000ULL);
         }
     }
-#ifdef TARGET_AMD64
-    return emitter::code_t(code | 0x4800000000ULL);
-#else
-    assert(!"UNREACHED");
-    return code;
-#endif
-}
+#ifdef TARGET_AMD64
+    return emitter::code_t(code | 0x4800000000ULL);
+#else
+    assert(!"UNREACHED");
+    return code;
+#endif
+}
```

Comment:
Do we know what changed as part of this diff? It looks identical to me, so maybe line endings or something else got changed by accident?

Reply:
I went over this and re-did the changes to make it cleaner but still looks weird. Not sure what's going on